apiserver/logsink.go

```diff
@@ -41,14 +41,13 @@ func newAgentLogWriteFunc(
 }
 
 func (s *agentLoggingStrategy) init(ctxt httpContext, req *http.Request) error {
-	st, entity, err := ctxt.stateForRequestAuthenticatedAgent(req)
+	authTag, err := ctxt.authenticatedAgentFromRequest(req)
```

Comment:
I'd have stuck with with the `ForRequest` convention used prior, which at least for me is more accurate conveyance than `FromRequest` you've adopted.